### PR TITLE
Asteroids improvements

### PR DESCRIPTION
**Pull request by hroncok**
- based on IRL conversation with @encukou
 - no window attribute (plenty of globals anyway, be consistent)
 - sometimes, be more specific
 - explain how the controls should work
 - center the anchor of images
 - add a bit info about CONSTANTS
 - flip sin and cos (just deal with Pyglet's rotation direction)
 - make sure .x, .y. and .rotation are propagated to .sprite
 - remove unnecessary asterisk
 - use  \<var\> where appropriate
 - add some missing commas
 - adding objects to objects can be done in \_\_init\_\_
   - so don't explicitly require people to do it manually
 - remove draw_circle once not needed anymore

### CODE REVIEW

**Comment by hroncok** [comment]
I spent ~1 hour trying to figure out the controls. The main changes are made where I found the materials confusing.

**Comment by hroncok** [comment]
cc @asgeirrr

File: lessons/projects/asteroids/index.md [hunk 1]
```diff
@@ -35,11 +35,11 @@ První krok bude naprogramovat vesmírnou loď, která půjde ovládat klávesni
 
 * Vesmírnou loď bude reprezentovat objekt třídy `Spaceship`.
 * Každá loď má vlastní atributy `x` a `y` (pozice),
-  `x_speed` a `y_speed` (rychlost), `rotation` (úhel natočení),
-  `sprite` (obrázek pro Pyglet) a `window` (okno, ve kterém se hraje).
+  `x_speed` a `y_speed` (rychlost), `rotation` (úhel natočení) a
+  `sprite` (obrázek pro Pyglet).
```
Comment (asgeirrr) [inline lessons/projects/asteroids/index.md]: Holky se často ptají, jaký je rozdíl mezi spritem a obrázkem - proč nevykreslujeme přímo obrázek, ale děláme vůbec nějakou instanci Spritu. Co sem napsat něco jako tohle?
```
`sprite` (2D objekt v Pygletu s polohou, rotací a obrázkem)
```

Comment (hroncok) [reply lessons/projects/asteroids/index.md]: To mi dává smysl. Díky

**Comment by encukou** [comment]
LGTM at first glance. With Oskar's review I';d say it's OK to merge.
I won't have time look over this until Sunday, but I can send a new PR with any adjustments.

**Pull request merged**